pkg/liquidity-mining/test/ChildChainGaugeTokenAdder.test.ts

```diff
@@ -35,6 +36,7 @@ describe('ChildChainGaugeTokenAdder', () => {
   sharedBeforeEach('deploy token', async () => {
     vault = await Vault.create({ admin });
     adaptor = vault.authorizerAdaptor;
```

Comment:
```suggestion
    const adaptor = vault.authorizerAdaptor;
```

ditto here as well. We could likely make this change in a number of the other test files.